contrib/autofs/files/automount

```diff
@@ -0,0 +1,4 @@
+# automount service
+
+type = process
+command = /usr/bin/automount -f
```

Comment:
pretty sure this is missing at least some dependencies to work

Reply:
i'd probably put it at early-fs-local.target here since this can run pretty early (at least after everything else is mounted)